src/API/Helpers/ApiClient.php

```diff
@@ -22,7 +22,10 @@ public static function setInfoHeadersData(InformationHeaders $infoHeadersData)
         self::$infoHeadersData = $infoHeadersData;
     }
 
-    public static function getInfoHeadersData()
+    /**
+     * @return InformationHeaders|null
+     */
+    public static function getInfoHeadersData() : ?InformationHeaders
```

Comment:
Worth adding some doc comments about the function itself as well?

Reply:
I mean ... yeah, probably 😆